# You can create copy of this file and load the required file on the rosparam server accordingly. Do not forget to modify the file name in the launch file.

#Need to enable one of the tracking algorithm
nearest_nbr: true
# Yet to implement
kalman: false

#-------------#
#Parameters for nearest neighbour filter, Best to use norm instead of individual error until you know what you what you are doing.
individual_error: false
# Tracking tolerance considered individually in x,y,z when 'Individual_error' is set to true
# Ideally tolerance shoould be greater than the (max robot speed in m/s)/(frame rate of motion capture system in Hz) 
# Current tolerance is set according to the 1m/s and 30FPS. But this should work in majority of the cases. Do not change until its necessary 
# in meters
E_x: 0.04
E_y: 0.04
E_z: 0.04
# Tracking tolerance considered as norm of 3D vector when 'Individual_error' is set to false
E: 0.04
#-------------#

# Add all the object names in this list to get the head count and the names of the objects
#object_names:
#  - crazyflie_1
#  - crazyflie_2
#   - rmtt_1

# Initial position of the object must be within the tracking tolerance (in meters).
# marker_config and orientation is not used as of now. 
#crazyflie_1:
#  marker_config: 0 
#  pose: 
#    position:
#      - 0.0
#      - 0.0
#      - 0.048
#    orientation:
#      - 0.0
#      - 0.0
#      - 0.0

#crazyflie_2:
#  marker_config: 0
#  pose: 
#    position:
#      - 0.0
#      - -1.75
#      - 0.042
#    orientation:
#      - 0.0
#      - 0.0
#      - 0.0

#rmtt_1:
#  marker_config: 0
#  pose:  
#    position:
#      - 0.0
#      - 0.0
#      - 0.094
#    orientation:
#      - 0.0
#      - 0.0
#      - 0.0

object_names:
  - cf1
cf1:
    marker_config: 0
    pose:
      orientation: []
      position:
      - -0.24554595351219177
      - -1.3885701894760132
      - 0.048130884766578674
